dev/README_RELEASE_PROVIDER_PACKAGES.md

```diff
@@ -1197,6 +1236,38 @@ do
  svn mv "${file}" "${base_file//rc[0-9]/}"
 done
 
+# Delete the old `pypi-rc` providers
+svn rm ${SOURCE_DIR}/pypi-rc/*
+```
```

Comment:
rm without commit?

Reply:
Yeah. It's next to impossible to describe it right without  actually trying it,  that's why we should do it together and update the docs when it's run.  It's more efficient than trying to make "dry-run" in your head.